bin/check-minio-update.rb

```diff
@@ -73,7 +73,7 @@ def get_latest_version(checkurl, platform)
   end
 
   def get_local_version
-    stdout, stderr, status = Open3.capture3('minio --version', unsetenv_others: true)
+    stdout, stderr, status = Open3.capture3({'PATH' => '/usr/local/bin:/usr/bin:/bin'}, 'minio --version', :unsetenv_others => true)
```

Comment:
```suggestion
    stdout, stderr, status = Open3.capture3({'PATH' => ENV['PATH']}, 'minio --version', :unsetenv_others => true)
```
Ich bin kein ruby-dev aber wie ist es damit ?